docusaurus/docs-internals/editor-controls/data-controls/toggle.md

```diff
@@ -0,0 +1,177 @@
+| `type`      | `string`                                 |    -    | Type should be `"switch"` to use this control                                                                                                                                                                                                                                                                                                                                                             |
+| `label?`    | `string`                                 |    -    | The label displayed on the left side of the control                                                                                                                                                                                                                                                                                                                                                       |
+| `position?` | `number`                                 |    -    | The position of the control in toolbar                                                                                                                                                                                                                                                                                                                                                                    |
+| `devices?`  | `"all"` \| `"desktop"` \| `"responsive"` | `"all"` | Define the devices where the control will be rendered. `"all"` renders the control on all devices. `"desktop"` renders the control only on desktop devices. `"responsive"` renders the control on both tablet and mobile devices.                                                                                                                                                                         |
+| `disabled?` | `boolean`                                | `false` | Configure the condition under which the control is disabled or enabled.                                                                                                                                                                                                                                                                                                                                   |
+| `choices?`  | `Array<Choice>`                          |  `[]`   | Choices is an array of objects, where each object represents a choice with a specific `title`, `value` , and `icon`. <br/><br/><b>`Choice: { title?: string, icon: string, value: string \| number }`</b> <br/><br/>`title`: The title of the element, which appears on hover.<br/>`icon`: The icon associated with the element.<br/>`value`: The value returned by the group when an option is selected. |
```

Comment:
1. why your `title` is not required ?
![image](https://github.com/EasyBrizy/Brizy-Local-Editor/assets/52622998/5b4744e6-0f40-4597-b1fb-6a56f3be20ce)

if in option is required

![image](https://github.com/EasyBrizy/Brizy-Local-Editor/assets/52622998/a2d2fd36-fb93-4f83-a3ce-5ae062480a3e)


2. and `value` set to be only string
3. "The value returned by the group when an option is selected." -> "The value returned by the control when this choice is selected."

P.S. check if gramatically correct